spec/music_album_spec.rb

```diff
@@ -0,0 +1,59 @@
+require './music_album'
+require './genre'
+require './label'
+require './author'
+
+RSpec.describe MusicAlbum do
+  before(:each) do
+    label1 = Label.new('Classic', 'grey')
+    label2 = Label.new('Hot', 'red')
+    genre = Genre.new(name: 'Country')
+    genre2 = Genre.new(name: 'Vintage')
+    author = Author.new('Mike', 'Smith')
+    author2 = Author.new('Jim', 'Ike')
+
+    @album1 = MusicAlbum.new(on_spotify: true, genre: genre, author: author, source: 'Bookstore',
+                             label: label1, publish_date: '1982-03-11')
+    @album2 = MusicAlbum.new(
+      on_spotify: false, genre: genre2, author: author2,
+      source: 'Radio', label: label2, publish_date: '1999-05-12'
+    )
+  end
+
+  it 'should create an instance of the MusicAlbum class' do
+    expect(MusicAlbum.all.empty?).to eq(false)
+    expect(@album1.on_spotify).to eq(true)
+  end
+
+  it 'lists all music albums' do
+    expect(MusicAlbum.all.length).to eq(4)
+  end
```

Comment:
- [ ] Kindly look into this test that is failing. **Note: line 55 to 57 failed as well**

<img width="449" alt="Untitled" src="https://github.com/Victoria129/ruby-capstone/assets/103386183/bcce175b-fe24-48e7-8ff5-2b781a4af54f">

Reply:
What happens with the test failure is that every time a new instance of the book is created with Rspec being called, the individual test files fail.

Fixing the issue now